rules/cross-platform/credential_access_cookies_chromium_browsers_debugging.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/12/21"
+maturity = "production"
+updated_date = "2020/12/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of a Chromium based browser with the debugging process argument, this may indicate an attempt
+to steal authentication cookies. An adversary may steal web application or service session cookies and use them to gain
+access web applications or Internet services as an authenticated user without needing credentials.
+"""
+false_positives = ["Developers performing browsers plugin or extension debugging."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+max_signals = 33
+name = "Potential Cookies Theft via Browser Debugging"
+references = [
+    "https://github.com/defaultnamehere/cookie_crimes",
+    "https://embracethered.com/blog/posts/2020/cookie-crimes-on-mirosoft-edge/",
+    "https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/post/multi/gather/chrome_cookies.md",
+    "https://posts.specterops.io/hands-in-the-cookie-jar-dumping-cookies-with-chromiums-remote-debugger-port-34c4f468844e",
+]
+risk_score = 47
+rule_id = "027ff9ea-85e7-42e3-99d2-bbb7069e02eb"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Windows", "macOS", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started", "info") and
+  process.name in (
+             "chrome.exe",
+             "Google Chrome",
+             "google-chrome-stable",
+             "google-chrome-beta",
+             "google-chrome",
+             "msedge.exe") and
+   wildcard(process.args, "--remote-debugging-port=*")
+'''
```

Comment:
```suggestion
query = '''
process where event.type in ("start", "process_started", "info") and
  process.name in (
             "chrome.exe",
             "Google Chrome",
             "google-chrome-stable",
             "google-chrome-beta",
             "google-chrome",
             "msedge.exe") and
   process.args : "--remote-debugging-port=*" and
   process.args : "--user-data-dir=*"
'''
```

Reply:
looks like `--remote-debugging-targets` and `--remote-debugging-pipe` may be able to [achieve this too](https://peter.sh/experiments/chromium-command-line-switches/). So maybe extend it to

```suggestion
query = '''
query = '''
process where event.type in ("start", "process_started", "info") and
  process.name in (
             "chrome.exe",
             "Google Chrome",
             "google-chrome-stable",
             "google-chrome-beta",
             "google-chrome",
             "msedge.exe") and
   process.args : ("--remote-debugging-port=*", "--remote-debugging-targets=*",  "--remote-debugging-pipe=*") and
   process.args : "--user-data-dir=*"
'''```